### PR TITLE
XW-94 use dns-prefetch

### PR DESCRIPTION
**Pull request by bognix**
https://wikia-inc.atlassian.net/browse/XW-94

### CODE REVIEW

File: server/views/_layouts/ember-main.hbs [hunk 1]
```diff
@@ -28,6 +28,18 @@
...
+		{{#if server.optimizelyScript}}
+			<link rel="dns-prefetch" href={{server.optimizelyScript}}>
+		{{/if}}
+		{{#unless queryParams.noexternals}}
+			<link rel="dns-prefetch" href="http://www.google-analytics.com/analytics.js">
+			<link rel="dns-prefetch" href=" https://beacon.wikia-services.com/__track/special/trackingevent">
+		{{/unless}}
+		{{#if qualarooScript}}
+			<link rel="dns-prefetch" href="https://geo.api.qualaroo.com/">
+			<link rel="dns-prefetch" href="http://s3.amazonaws.com/ki.js/52510/bgJ.js">
```
Comment (rogatty) [inline server/views/_layouts/ember-main.hbs]: This one is already available in localSettings.

File: server/views/_layouts/ember-main.hbs [hunk 2]
```diff
@@ -28,6 +28,18 @@
...
 
+		{{#if server.optimizelyScript}}
+			<link rel="dns-prefetch" href={{server.optimizelyScript}}>
+		{{/if}}
+		{{#unless queryParams.noexternals}}
+			<link rel="dns-prefetch" href="http://www.google-analytics.com/analytics.js">
+			<link rel="dns-prefetch" href=" https://beacon.wikia-services.com/__track/special/trackingevent">
+		{{/unless}}
+		{{#if qualarooScript}}
+			<link rel="dns-prefetch" href="https://geo.api.qualaroo.com/">
```
Comment (rogatty) [inline server/views/_layouts/ember-main.hbs]: We don't have it anywhere in the code. Isn't it some internal script called by the one below? If yes then we should treat it as a "private API".

File: server/views/_layouts/ember-main.hbs [hunk 3]
```diff
@@ -28,6 +28,18 @@
 			<!-- endbuild -->
 		{{/if}}
 
+		{{#if server.optimizelyScript}}
+			<link rel="dns-prefetch" href={{server.optimizelyScript}}>
+		{{/if}}
+		{{#unless queryParams.noexternals}}
+			<link rel="dns-prefetch" href="http://www.google-analytics.com/analytics.js">
```
Comment (rogatty) [inline server/views/_layouts/ember-main.hbs]: It's duplicated in tracking.hbs, how about moving it to localSettings?

File: server/views/_layouts/ember-main.hbs [hunk 4]
```diff
@@ -28,6 +28,18 @@
...
+			<link rel="dns-prefetch" href=" https://beacon.wikia-services.com/__track/special/trackingevent">
```
Comment (rogatty) [inline server/views/_layouts/ember-main.hbs]: This is called from mercury/modules/Trackers/Internal, only after Ember is loaded. How is this change good for our users?

File: config/localSettings.base.js [hunk 5]
```diff
@@ -212,7 +213,8 @@ const localSettings = {
 		},
 		krux: {
 			mobileId: 'JTKzTN3f'
-		}
+		},
+		gaUrl: '//www.google-analytics.com/analytics.js'
```
Comment (rogatty) [inline config/localSettings.base.js]: I'd move it to `ua` above, it's the same thing.

File: server/views/_layouts/ember-main.hbs [hunk 6]
```diff
@@ -28,6 +28,16 @@
 			<!-- endbuild -->
 		{{/if}}
 
+		{{#if server.optimizelyScript}}
+			<link rel="dns-prefetch" href={{server.optimizelyScript}}>
```
Comment (rogatty) [inline server/views/_layouts/ember-main.hbs]: I think we should use double quotes.

File: server/views/_layouts/ember-main.hbs [hunk 7]
```diff
@@ -28,6 +28,16 @@
 			<!-- endbuild -->
 		{{/if}}
 
+		{{#if server.optimizelyScript}}
```
Comment (rogatty) [inline server/views/_layouts/ember-main.hbs]: Can we move it all to a partial named `prefetch.hbs`?

**Comment by rogatty** [comment]
LGTM

**Comment by vforge** [comment]
LGTM

**Pull request merged**